src/main/web/src/organisms/login/Login.tsx

```diff
@@ -121,17 +137,14 @@ const Login: React.FC<Props> = () => {
                             </div>
 
                             <div className="google-oauth-login">
-                                <GoogleOAuthProvider clientId="">
-                                    <GoogleLogin size={'medium'} logo_alignment={'center'} ux_mode={'popup'} useOneTap={true}
-                                                 text={"continue_with"}
-                                                 onSuccess={credentialResponse => {
-                                                     console.log(credentialResponse);
-                                                 }}
-                                                 onError={() => {
-                                                     console.log('Login Failed');
-                                                 }}
-                                    />
-                                </GoogleOAuthProvider>
+
+                              <GoogleLogin size={'medium'} logo_alignment={'center'} ux_mode={'popup'} useOneTap={true}
+                                             text={"continue_with"}
+                                    onSuccess={handleGoogleLogin}
+                                    onError={() => {
+                                        console.log('Login Failed');
+                                    }}
+                                />
```

Comment:
Kann GoogleOAuthProvider einfach wieder gelöscht werden?

Reply:
Ja, der ist an der Stelle nicht nötig. Wird bereits in App.tsx genutzt